ttnn/cpp/ttnn/core.cpp

```diff
@@ -7,18 +7,18 @@
 
 namespace ttnn {
 
-std::atomic<std::int64_t> TENSOR_ID = -1;
-std::atomic<std::int64_t> PYTHON_OPERATION_ID = -1;
-std::atomic<std::int64_t> DEVICE_OPERATION_ID = -1;
+std::atomic<std::int64_t> TENSOR_ID = 0;
```

Comment:
Nice fix!
could you create singleton and put all global variables inside so there is no direct access from the outside?
I saw this code and that was my plan.

Reply:
Sure, does calling the singleton `CoreIDs` make sense?

Reply:
CoreIDs or GlobalCoreIDs sounds good for me.